### PR TITLE
fozzie-components@v3.25.0 Fix Fozzie docs link in README

### PR DESCRIPTION
**Pull request by adammorr**
Updates to Fozzie docs link in the readme

### CODE REVIEW

**Review by DamianMullins — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "3.24.0",
+  "version": "3.25.0",
```
Comment (DamianMullins) [inline package.json:3]: As this is a readme change I'd have said a #trivial PR or patch version would suffice here but happy to approve in any case.

**Review by DamianMullins — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (adammorr) [reply package.json:3]: @DamianMullins thanks for pointing that out, I didn't realise. Happy to change this, let me know!

Comment (DamianMullins) [reply package.json:3]: Probably best to stick with the conventions so updating it would be ideal 👍🏻

Comment (dandel10n) [reply package.json:3]: Hm, I think that updating readme is a #trivial change and should go without version bump as for end users there is no changes, right?

**Comment by dandel10n** [comment]
Closed as the issue has been fixed

File: package.json (hunk 1 above)
Comment (DamianMullins) [reply package.json:3]: Agree, remove the versioning and stick a #trivial prefic in the PR title @adammorr

**Pull request closed without merging**